### PR TITLE
fix: unique name for add template

### PR DESCRIPTION
**Pull request by eddiejaoude**
closes #ISSUE-NO

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/add-api-user.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: EddieHub API
+name: EddieHub API add
```
Comment (kb-s) [inline .github/ISSUE_TEMPLATE/add-api-user.yml:1]: can we write it as **"name:  add EddieHub API"**

**Review by kb-s — CHANGES_REQUESTED** [review]

**Review by vijay-jaisankar — COMMENTED** [review]
In my opinion, `Eddiehub API` is an apt name for this operation if we treat it to be the most atomic operation possible.

If there is scope for similar operations in the future, naming this something like `Add User to Eddiehub API` would make more sense.

**Comment by eddiejaoude** [comment]
Yes we have a delete template now too, so we needed to be more specific as there was a name clash

File: .github/ISSUE_TEMPLATE/add-api-user.yml (hunk 1 above)
Comment (krshkodes) [inline .github/ISSUE_TEMPLATE/add-api-user.yml:1]: I would suggest this 🤔 

```suggestion
name: Get Added to the EddieHub API
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by krshkodes — CHANGES_REQUESTED** [review]

**Review by bishtanuj — APPROVED** [review]

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by vijay-jaisankar — APPROVED** [review]

**Comment by kb-s** [comment]
# Looking good to me 
@krshkodes *your suggestion is awesome*

**Comment by kb-s** [comment]
*Thanks @vijay-jaisankar for reviewing it*

**Pull request merged**